Adding a blog on Falco performance testing with event-generator (#370)
falcosecurity/falco-website: content/en/blog/Falco-Performance-Testing.md

> |           Title           |                         Description                          |    Date    | Author         |
> | :-----------------------: | :----------------------------------------------------------: | :--------: | -------------- |
> | Falco Performance Testing | Understanding Falco performance testing with event-generator | 2021-01-20 | Samar Sidharth |
> 

Reviewer: ```suggestion
---
title: Falco Performance Testing
description: Understanding Falco performance testing with event-generator
date: 2020-09-23
author: 2021-01-20
slug: falco-performance-testing
---

```

# Falco Performance Testing

> **Special Thanks to Leonardo Grasso from Sysdig for assisting me**

Reviewer: ```suggestion
**Special Thanks to [Leonardo Grasso](https://github.com/leogr) for assisting me**

```
:heart:

  INFO statistics                                    cpu="21.9%" lost="0%" res_mem="121 MB" throughput="1017.4 EPS" virt_mem="1.5 GB"
  INFO
  INFO round #13                                     sleep="48.828µs"
  INFO resting...
  INFO syscall.WriteBelowBinaryDir                   actual=1787 expected=1787 ratio=1
  INFO syscall.ChangeThreadNamespace                 actual=1788 expected=1788 ratio=1
  INFO syscall.ReadSensitiveFileUntrusted            actual=1786 expected=1786 ratio=1
  INFO statistics                                    cpu="23.6%" lost="0%" res_mem="121 MB" throughput="1072.2 EPS" virt_mem="1.5 GB"
  INFO
  INFO round #14                                     sleep="24.414µs"
  INFO resting...
  INFO syscall.WriteBelowBinaryDir                   actual=1614 expected=1614 ratio=1
  INFO syscall.ChangeThreadNamespace                 actual=1613 expected=1613 ratio=1
  INFO syscall.ReadSensitiveFileUntrusted            actual=1615 expected=1615 ratio=1
  INFO statistics                                    cpu="22.4%" lost="0%" res_mem="121 MB" throughput="968.4 EPS" virt_mem="1.5 GB"
  
  ```

> 
> 
> 
> 

Reviewer: ```suggestion
```

Author: Thanks @leogr :)
